packages/serialize/types/index.d.ts

```diff
@@ -13,7 +17,7 @@ export type CSSPropertiesWithMultiValues = {
     | Array<Extract<CSSProperties[K], string>>
 }
 
-export type CSSPseudos = { [K in CSS.Pseudos]?: CSSObject }
+export type CSSPseudos = { [K in CSS.Pseudos]?: CSSObject | SerializedStyles }
```

Comment:
One of the thing I noticed while using this lib is that there is a test for `css composition` but it doesn't cater that in type.

Reply:
Are there reasons why the following convention is not allowed even though they worked?
```js
css(() => '')
css`${() => ''}`
```